### PR TITLE
Added variable to control wpilib version easier.

### PR DESCRIPTION
**Pull request by thenetworkgrinch**
The version variable is purely there to make it easier to control the WPILib version in a centralized manner.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,6 +8,9 @@ plugins {
   id 'edu.wpi.first.GradleVsCode' version '2.1.0'
 }
 
+// WPILib Version
+ext.wpilibVersion = "$wpilibVersion"
```
Comment (rzblue) [inline build.gradle]: This needs to actually have a value

**Review by rzblue — CHANGES_REQUESTED** [review]

File: config.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ nativeUtils.withCrossLinuxArm64()
 nativeUtils {
     wpi {
         configureDependencies {
-            wpiVersion = "2025.+"
+            wpiVersion = "$wpilibVersion"
```
Comment (ThadHouse) [inline config.gradle]: No need to restringify this here. Just assign directly.

**Review by ThadHouse — APPROVED** [review]

File: config.gradle [hunk 3]
```diff
@@ -8,7 +8,7 @@ nativeUtils.withCrossLinuxArm64()
 nativeUtils {
     wpi {
         configureDependencies {
-            wpiVersion = "2025.+"
+            wpiVersion = wpilibVersion
             opencvYear = "frc2024"
```
Comment (ThadHouse) [inline config.gradle:12]: Can you also add the same things for opencvYear and opencvVersion? That way the versions that are shared are in a common place.

Comment (thenetworkgrinch) [reply config.gradle:12]: Yes, but that was out of scope of this PR

**Pull request merged**